susemanager-utils/testing/docker/scripts/generate-reportdb-docs.sh

```diff
@@ -99,33 +99,41 @@ CONFIG_FILE=/root/reportdb-docs.properties
 
 OUTPUT_FILE=reportdb-schema-docs.tar.xz
 if [ -f $OUTPUT_FILE ]; then
-    echo Removing previous generated tarball
+    echo "Removing previous generated tarball $OUTPUT_FILE"
     rm -f $OUTPUT_FILE
 fi
 
-OUTPUT_DIR=$(cat $CONFIG_FILE | grep schemaspy.o | cut -c 13-)
-if [ -d $OUTPUT_DIR ]; then
-    echo Removing previous generated docs at $OUTPUT_DIR
-    rm -rf $OUTPUT_DIR
+OUTPUT_DIR=$(grep schemaspy.o $CONFIG_FILE | cut -c 13-)
+if [ -d "$OUTPUT_DIR" ]; then
+    echo "Removing previous generated docs at $OUTPUT_DIR"
+    rm -rf "$OUTPUT_DIR"
 fi
 
 SCHEMASPY_REPO=mackdk/schemaspy
 SCHEMASPY_VERSION=6.1.1
 SCHEMASPY_JAR=/root/schemaspy.jar
 if [ ! -f $SCHEMASPY_JAR ]; then
-    echo Retrieving SchemaSpy version $SCHEMASPY_VERSION
+    echo "Retrieving SchemaSpy version $SCHEMASPY_VERSION"
     wget -q --show-progress "https://github.com/$SCHEMASPY_REPO/releases/download/v$SCHEMASPY_VERSION/schemaspy-$SCHEMASPY_VERSION.jar" -O $SCHEMASPY_JAR
 fi
 
-java -cp $(build-classpath ongres-scram) -jar $SCHEMASPY_JAR -configFile $CONFIG_FILE -dp $(build-classpath postgresql-jdbc) -label "$BRAND_NAME Reporting"
+# Check postgresql runtime dependency. If version is >= 42.2.19, we need ongres as well
+POSTGRESQL_VERSION=$(rpm -q --qf '%{V}\n' postgresql-jdbc)
+if echo -e "$POSTGRESQL_VERSION\n42.2.19" | sort --reverse --version-sort --check=quiet; then
+    DRIVER_PATH=$(build-classpath postgresql-jdbc ongres-scram ongres-stringprep)
+else
+    DRIVER_PATH=$(build-classpath postgresql-jdbc)
```

Comment:
Hmm, I think ongres-scram is also needed for the older version. Just stringprep is only needed for newer versions.
ongres-scram can only be ignored when the DB auth algorithm is md5. As soon as it is scram-sha256, you need scram.
So maybe you can check pg_hba.conf

Reply:
That's not what I'm seeing from the execution. In the container, everything is defined as `md5`, but still the newer postgresql version requires those two additional jars to work. Anyway I will have an additional look.